ui/src/views/PageNotFoundView.vue

```diff
@@ -6,12 +6,19 @@
   >
     <v-row justify="center">
       <v-col cols="auto">
-        page not found
+        <h1>Page not found</h1>
+        <p>The page either doesn't exist or you don't have access to this page</p>
+        <center>
+          <v-img :src="mainPicture" :width="imageWidth" contain />
+        </center>
         <!-- to do: check the log in state, redirect based on the user type -->
         <v-btn
           depressed
           large
           to="/"
+          :color="variables.default.primary"
+          :width="imageWidth"
```

Comment:
Is there any reason on why the width props is bound to the data? Cant the value be string  (width="100") for example

Also the var name is imageWidth while the element is button

Reply:
Also, have you checked the mobile view? (Use mobile dev tools in dev tools, there is a phone icon near the top right screen, click on it and you can choose the device).

Reply:
I will put all those elements in a `col` and just set the width for that `col`. Also yes I did check the mobile view on various devices available in the dev tools.

Reply:
Great, thank you! I will initialize the variables as global variables, as for now please wait or probably work on some other things/cards..